public/js/views/purchase.jsx

```diff
@@ -1,44 +1,55 @@
-'use strict';
+import React, { Component, PropTypes } from 'react';
 
-var React = require('react');
-var bindActionCreators = require('redux').bindActionCreators;
-var Connector = require('redux/react').Connector;
+import { bindActionCreators } from 'redux';
+import { Connector } from 'redux/react';
 
-var CardDetails = require('views/card-details');
-var CardListing = require('views/card-listing');
-var CompletePayment = require('views/complete-payment');
-var purchaseActions = require('actions/purchase');
+import * as purchaseActions from 'actions/purchase';
 
+import CardDetails from 'views/card-details';
+import CardListing from 'views/card-listing';
+import CompletePayment from 'views/complete-payment';
 
-module.exports = React.createClass({
 
-  displayName: 'Purchase',
+export default class Purchase extends Component {
 
-  propTypes: {
-    productId: React.PropTypes.string.isRequired,
-    user: React.PropTypes.object.isRequired,
-  },
+  static propTypes = {
+    CardDetails: PropTypes.object,
+    CardListing: PropTypes.object,
+    CompletePayment: PropTypes.object,
+    productId: PropTypes.string.isRequired,
+    user: PropTypes.object.isRequired,
+  }
 
-  selectData: function(state) {
+  static defaultProps = {
+    CardDetails: CardDetails,
```

Comment:
I think you should do the same thing here to avoid underscores in render: `CardDetails: DefaultCardDetails`